src/index.html

```diff
@@ -1,14 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <a href="" class="logo_image">
+        <img src="/images/logo.png" alt="" >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav_item">
+            <a href="#"
+               class="nav_link"
+               data-qa="hover"
+            >
```

Comment:
коже атрибут має починатись з нової ліній, включно з першим
```suggestion
            <a
              href="#"
              class="nav_link"
              data-qa="hover"
            >
```